core/src/avm2/globals/flash/display/NativeMenuItem.as

```diff
@@ -1,6 +1,32 @@
 package flash.display {
     import flash.events.EventDispatcher;
+    import __ruffle__.stub_constructor;
+    import __ruffle__.stub_getter;
+
+    [API("667")]
```

Comment:
nit: It might be worth a little comment that the docs say `661` here (AIR 1.0), but it's not super important, and definitely not a blocker.

Reply:
(and `swc` here too)